gym/wrappers/rescale_action.py

```diff
@@ -42,6 +59,14 @@ def __init__(self, env, min_action, max_action):
         )
 
     def action(self, action):
+        """Rescales the action affinely from  [:attr:`min_action`, :attr:`max_action`] to the action space of the wrapped environment, :attr:`env`.
```

Comment:
An ambiguity I may have introduced myself. It's unclear whether "wrapped environment" is the environment that is being wrapped or the base environment + wrapper.
 
```suggestion
        """Rescales the action affinely from  [:attr:`min_action`, :attr:`max_action`] to the action space of the base environment, :attr:`env`.
```